### ISSUE TITLE
Auth fails silently

### ISSUE THREAD

**Issue by jpstevens** [hapijs/bell#163]
Is there an easy way to enable more robust error logging in Bell?

Any misconfiguration in my custom provider simply returns:
```
{
statusCode: 500,
error: "Internal Server Error",
message: "An internal server error occurred"
}
```

With nothing written to stdout/stderr.

Please help, I need my logs... :broken_heart:

**Comment by ldesplat** [comment]
Yes, I agree there should be more robust error logging. You can see #158 for a discussion on it. If you want it done sooner rather than later, I will gladly accept a PR for it.

**Comment by ldesplat** [comment]
Fixed in 705303f
